app/recordtransfer/templates/includes/delete_in_progress_submission_modal.html

```diff
@@ -0,0 +1,24 @@
+{% extends "includes/base_modal.html" %}
+{% load i18n %}
+{% block modal_id %}
+    delete_in_progress_submission_modal
+{% endblock modal_id %}
+{% block modal_title %}
+    {% trans "Are you sure you want to delete this in-progress submission?" %}
+{% endblock modal_title %}
+{% block modal_content %}
+    <p>
+        {% trans "Do you wish to delete the in-progress submission from" %} {{ in_progress.last_updated|date:"F j, Y g:ia" }}
+        {% if in_progress.title %}titled "{{ in_progress.title }}"{% endif %}
+        ?
```

Comment:
This is split up in a way that makes translation difficult. How about changing it to this?

```suggestion
        {% if in_progress.title %}
            {% blocktrans with title=in_progress.title date=in_progress.last_updated|date:"F j, Y g:ia" %}
            Do you wish to delete the in-progress submission from {{ date }} titled "{{ title }}"?
            {% endblocktrans %}
        {% else %}
            {% blocktrans with date=in_progress.last_updated|date:"F j, Y g:ia" %}
            Do you wish to delete the in-progress submission from {{ date }}?
            {% endblocktrans %}
        {% endif %}
```